src/main/webapp/profilePage.js

```diff
@@ -0,0 +1,92 @@
+var experience;
+var name;
+var careerPath;
+
+function onLoadFunctions() {
+  fetchBlobstoreUrl();
+  getEquippedAccessories();
+  getAccessories();
+  fillText();
+}
+
+function fillText() {
+  getName();
+  getExperiencePoints();
+  getCareerPath();
+}
+
+function getName() {
+//   fetch("/name")
```

Comment:
Change to "get-player-name", then un-comment to integrate with Many's (now-merged) code.

Reply:
+1 would pull master